frontend/src/views/edicoesEmLote/EdicoesEmLoteResumo.vue

```diff
@@ -41,22 +45,30 @@ onMounted(() => {
   <CabecalhoDePagina />
 
   <section class="edicoes-em-lote-resumo">
-    <h3 class="edicoes-em-lote-resumo__titulo">
-      Edição
-    </h3>
-
-    <dl
-      v-if="emFoco"
-      class="edicoes-em-lote-resumo__detalhes"
-    >
-      <div
-        v-for="(itemDetalhe, detalheIndex) in detalhesEdicao"
-        :key="`detalhe-item--${detalheIndex}`"
+    <article>
+      <template
```

Comment:
Se for tudo responsivo, você pode fazer o loop na `<div>`. Seu pensamento está muito "bootstrap", @GustavoFSoares